src/components/MCQ/MCQStatus.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <!-- ToDo: add quizstatus report card here -->
+    {{ quizStatus }}
```

Comment:
I would probably take this out for the sake of presentation and add it in the next ticket.

Odd to see json just on the page and we're getting the score anyways.

Reply:
fixed